modules/simple_forms_api/spec/fixtures/form_json/vba_20_10207-third-party-non-veteran.json

```diff
@@ -40,4 +41,4 @@
       "start_date": "2018-01-01"
     }
   ]
-}
+}
```

Comment:
Does your text editor automatically remove blank lines at the end of some files? a newline at the end is usually preferred. Can you modify a setting to change that so a newline is added (or at least not removed)?